packages/lesswrong/components/review/UserReviews.tsx

```diff
@@ -17,25 +17,46 @@ const UserReviews = () => {
 
   if (loading) return <Loading />
   if (!user) return <Error404 />
+  if (!year) return <Error404 />
+
+  let nominationsView
+  let reviewsView
+
+  switch (year) {
```

Comment:
Note: On one hand, at some point I'll probably condense "nominations2018", "nominations2019", etc into a single "nominations" view that takes a year parameter. But, I'm not 100% sure about that because I might actually be getting rid of nominations this year and just using "positive reviews" in their place, and I want to think about how the overall review is shaping up before doing refactors.